deploy/operator/role.yaml

```diff
@@ -53,12 +53,27 @@ rules:
   - statefulsets
   verbs:
   - '*'
+- apiGroups:
+  - console.openshift.io
+  resources:
+  - consoleplugins
+  verbs:
+  - '*'
 - apiGroups:
   - nmstate.io
   resources:
   - '*'
   verbs:
   - '*'
+- apiGroups:
+  - operator.openshift.io
+  resources:
+  - consoles
+  verbs:
+  - get
+  - list
+  - update
+  - watch
```

Comment:
This things has to be present only if openshift is detected, we don't want to polute non openshift deploys.

Reply:
We're using kubebuilder markers (which are comments) to generate the RBAC roles.
How could it be possible to configure conditionally this way?

Reply:
Nope, let's just leave like that